doc/developer-guides/sw_design_guidelines.rst

```diff
@@ -73,6 +75,12 @@ below:
 
   - Input data which is only specified by external safety application.
 
+.. note:: If input data can be specified by both of non-safety VM and safety VM,
```

Comment:
```suggestion
.. note:: If input data can be specified by both a non-safety VM and a safety VM,
```

Reply:
Thanks for your comment and review, will update it.